pkg/controller/seed-controller-manager/openshift/openshift_controller_test.go

```diff
@@ -142,7 +142,7 @@ func TestResources(t *testing.T) {
 				},
 				&corev1.Service{
 					ObjectMeta: metav1.ObjectMeta{
-						Name:      "apiserver-external",
+						Name:      "apiserver",
```

Comment:
Service name inconsistency compared to the K8s one.

Reply:
That's just the name used in the test so it's harmless, but it's a good starting point for a deeper reflection. It comes from a previous attempt I made where I renamed `apiserver-external` to `apiserver`. This probably makes more sense, but it requires some clean-up logic to avoid port clashes. After better thinking it could be worthy to have it.